src/main/java/org/maxgamer/rs/structure/ReflectUtil.java

```diff
@@ -4,72 +4,62 @@
 import java.lang.reflect.Modifier;
 import java.util.Arrays;
 
-public class ReflectUtil{
-	public static String describe(Object o){
-		StringBuilder result = new StringBuilder();
-		String newLine = System.getProperty("line.separator");
+public class ReflectUtil {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1118)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AUndocumentedApi)